.github/workflows/build.yaml

```
@@ -0,0 +1,72 @@
+on:
+  # to trigger build when daily tinytex build is finished
+  repository_dispatch:
+    types: [daily-build]
+  workflow_dispatch:
+
+    
+name: Build Daily TinyTeX Full Bundles
+
+env:
+  TINYTEX_INSTALLER: TinyTeX
+  GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  GH_REPO: cderv/tinytex-releases
```

- [ ] Change repo before merging

---

I think it's better to use `github.repository`
```suggestion
  GH_REPO: ${{ github.repository }}
```

---

Thanks. I'll switch to that before merging. I am hard coding the value here because I want to be absolutely sure that if this workflow runs, it is in my fork until the work is ready. Even if by mistake it is run from this repo. 

But I'll probably switch to your suggestion - I am not even sure I need to specific `GH_REPO` as here I am running `gh` is the same repo I want to target (which is sometimes not the case hence the `GH_REPO` option set)

thanks!